orchagent/routeorch.h

```diff
@@ -132,6 +150,7 @@ class RouteOrch : public Orch, public Subject
 
     RouteTables m_syncdRoutes;
     NextHopGroupTable m_syncdNextHopGroups;
+    NextHopRouteTable m_nextHops;
```

Comment:
it is better to explain what does the nexthop router table store? the nexthop -> route map? does this includes all the routes to T1?

Reply:
It is mentioned at line 67. This is generic to be useful for other use-cases